recipe/meta.yaml

```
@@ -11,6 +13,12 @@ package:
 source:
   url: https://github.com/single-cell-data/TileDB-SOMA/archive/{{ version }}.tar.gz
   sha256: {{ sha256 }}
+# Pre-release canary "will Conda be green if we release":
+# source:
+#   git_url: https://github.com/single-cell-data/TileDB-SOMA.git
+#   git_rev: 00commithashyouwanttotest00commithashyouwanttotest00
+#   git_depth: 1
+
 
 build:
   number: 1
```

When you bump the version, you need to move the build number back to `0`
```suggestion
  number: 0
```

---

Wow OK TIL @jdblischak -- I'll update my notes -- thank you for info-sharing!